tr1d1um.yaml

```diff
@@ -123,6 +123,46 @@ log:
 # information using Argus.
 # Optional: if key is not supplied, webhooks would be disabled.
 webhook:
+
+  # validation provides options for validating the webhook's URL and TTL.
```

Comment:
We should add information on which fields are required to be added in the config file and if not what the default values are